ogticrd/covid-safe-paths
# Health authorities

![Screenshot_20200603-155553](https://user-images.githubusercontent.com/3283020/83693140-73e7d380-a5c3-11ea-9811-52ec01885239.png)
![Screenshot_20200603-155538](https://user-images.githubusercontent.com/3283020/83693155-7fd39580-a5c3-11ea-82b8-4ba2c1e074ca.png)



https://trello.com/c/0WZekEYC/7-as-a-positive-patients-i-want-my-path-history-so-that-i-can-keep-people-around-me-safe

#### Description:

As a positive patients, I want my path history so that I can keep people around me safe



#### Linked issues:



#### Screenshots:



#### How to test:
This is mostly backend changes that are not going to be testable. 

Exposed API it's working but you need to ask for critical points or change on request to have a match. 

Also on first entry to LocationTracking Screen a new pop notification appears about health authorities and you can check your health authorities on settings
